docs/dm/zh_CN/dm-worker-unit.md

```diff
@@ -0,0 +1,69 @@
+DM-Worker 详细介绍
```

Comment:
should we need unify case (upper / lower) for component / unit names?

Reply:
still not do it